test/api/models/document.js

```diff
@@ -43,8 +55,30 @@ describe('Document', () => {
         return factory.create('file', { text })
           .then((file) => factory.create('documentWithFile', { file_id: file.attributes.id }))
           .then((doc) => new Document({ id: doc.attributes.id }).fetch({ withRelated: ['file'] }))
-          .then((doc) => should(doc.toJSON().text).equal(text));
-      })
+          .then((doc) => should(doc.toJSONSummary().text).equal(text));
+      });
+    });
+  });
+
+  describe('serialize', () => {
+    it(`doesn\'t return virtuals`, () => {
+      const dc_id = '1000-the-file';
+
+      return factory.create('file', { documentcloud_id: dc_id })
+        .then((file) => factory.create('documentWithFile', { file_id: file.attributes.id }))
+        .then((doc) => new Document({ id: doc.attributes.id }).fetch({ withRelated: ['file'] }))
+        .then((doc) => should(doc.toJSON().documentcloud_id).be.undefined());
+    });
+    it('returns summarized version of file', () => {
+      return factory.create('file')
+        .then((file) => factory.create('documentWithFile', { file_id: file.attributes.id }))
+        .then((doc) => new Document({ id: doc.attributes.id }).fetch({ withRelated: ['file'] }))
+        .then((doc) => doc.toJSON().file.should.deepEqual(doc.related('file').toJSONSummary()));
+    });
+    it('returns empty object if it lacks file', () => {
+      return factory.create('document', {file_id: undefined})
+        .then((doc) => new Document({ id: doc.attributes.id }).fetch({ withRelated: ['file'] }))
+        .then((doc) => doc.toJSON().file.should.deepEqual({}));
```

Comment:
I should've used `equal()` instead of `deepEqual()` but for some reason that raised: `AssertionError: expected Object {} to be Object {}` :smile:.